docs/pages/reference/backends.mdx

```diff
@@ -620,6 +620,182 @@ Config option priority is applied in the following order:
   are not supported in certain regions or environments or are only supported in GovCloud.
 </Admonition>
 
+## Athena
+
+If you are running Teleport on AWS, you can use
+[Athena](https://aws.amazon.com/athena/) based audit log system that uses
+[Parquet files](https://parquet.apache.org/) stored on
+[S3](https://aws.amazon.com/s3/) as a storage back-end to achieve High
+Availability. Athena backend supports only one type of Teleport data:
+
+- Audit log events
+
+The athena audit logs backend provides better scalability and searchability in
+comparison to DynamoDB.
+
+Athena audit logs are eventually consistent, this means that events won't be
+immediately displayed in UI. It can take up to 1 minute until they are
+visible (depending on `batchMaxInterval` setting and events load).
+
+### Infrastructure set up
+
+Teleport uses AWS SNS and SQS subscribed to topic, for publishing events.
```

Comment:
What does "Teleport uses SQS subscribed to topic" mean?

Reply:
> The Auth service uses an SQS queue subscribed to an SNS topic for event
publishing.

does it sounds better?